website/pypi.md

```diff
@@ -22,7 +22,7 @@ The command below provides access to the core functionalities behind the [OpenBB
 pip install openbb-terminal
 ```
 
-If you wish to utilize our **Portfolio Optimization** or **Machine Learning / Artificial Intelligence** toolkits, please see instructions [here](https://docs.openbb.co/sdk/installation).
+If you wish to utilize our **Portfolio Optimization** or **Machine Learning / Artificial Intelligence** toolkits, please see instructions [here](https://docs.openbb.co/platform/installation).
```

Comment:
Those toolkits are not yet supported on the Platform, should we still point this section to the platform installation procedure?

Reply:
good catch. maybe just pointing to the docs root?

Reply:
reverted